pages/Enhanced-Crafting-Table.md

```diff
@@ -0,0 +1,26 @@
+# Enhanced Crafting Table
```

Comment:
Wiki pages should not include headers.
They have headers automatically, this would create two.